integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.5.2
+0.5.3
```

Comment:
Version updated  yet again with some nrf fixes :(

Could you move to 0.5.4?

Reply:
Yes, sure. But if there are no other comments, can we ask for a merge? :)